components/loading-box/src/stories.js

```diff
@@ -119,3 +119,30 @@ stories.add('LoadingBox (long)', () => (
     </div>
   </LoadingBox>
 ));
+
+stories.add('with children that have short height (minHeight 100px)', () => (
+  <LoadingBox
+    loading={boolean('loading', false)}
+    backgroundColor={text('backgroundColor', '#fff')}
+    timeIn={number('timeIn', 800)}
+    timeOut={number('timeOut', 200)}
+    backgroundColorOpacity={number('backgroundColorOpacity', 0.85)}
+    spinnerColor={text('spinnerColor', '#000')}
+  >
+    Lorem ipsum dolor sit amet
+  </LoadingBox>
+));
+
+stories.add('with bolded texts', () => (
+  <LoadingBox
```

Comment:
minor and up to you - I don't think we need knobs for this story, just force loading = true and remove all other props so that they use defaults

Reply:
ok